underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -163,6 +163,10 @@ public OutputStream create(String path) throws IOException {
   public OutputStream create(String path, CreateOptions options) throws IOException {
     LOG.debug("Create method: {}", path);
 
+    if (!path.startsWith(mContainerPrefix)) {
+      path = mContainerPrefix + CommonUtils.stripPrefixIfPresent(path, PATH_SEPARATOR);
+    }
+
```

Comment:
Persist operation with a Swift UFS was failing with a NullPointerException. The cause was that the SwiftUnderFileSystem.create was expecting a path of the form 'swift://swift-container/path_to_file' but was called with path as '/path_to_file'.